app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/usecase/LoadLocalCharactersUseCase.kt

```diff
@@ -0,0 +1,8 @@
+package com.puzzlebench.clean_marvel_kotlin.domain.usecase
+
+import com.puzzlebench.clean_marvel_kotlin.data.service.LoadLocalCharactersImpl
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+
+class LoadLocalCharactersUseCase(private val loadLocalCharacterImpl: LoadLocalCharactersImpl) {
```

Comment:
1-. You should have the **LoadLocalCharactersImpl** here local.. do not pass it for parameter..
2-. Use the general type, not the implementation

Reply:
Done!